src/simulation/sensors/camera/camera.cpp

```diff
@@ -331,9 +333,30 @@ void Camera::UpdateState(uint64_t currentSimNanos)
     cameraMsg.ppFocusDistance = this->ppFocusDistance;
     cameraMsg.ppMaxBlurSize = this->ppMaxBlurSize;
 
+    /*! - Populate the camera model message */
+    cameraModelMsg.cameraID = this->cameraIdentification;
+    strcpy(cameraModelMsg.parentName, this->parentSpacecraftName.c_str());
+    cameraModelMsg.resolution[0] = this->resolution[0];
+    cameraModelMsg.resolution[1] = this->resolution[1];
+    cameraModelMsg.renderRate = this->imageCadence;
```

Comment:
:large_orange_diamond: I'm possibly missing the point (and will dig further), but here we set `imageCadence` to `renderRate` in the message, yet the module also has a `renderRate` variable. Is this because we're placing these functionalities alongside each before we come back and significantly refactor this module in general?

Reply:
That's correct, almost all of the camera parameters are duplicated in order to keep the previous functionality. The camera refactor ticket is alive and well!